### ISSUE TITLE
I don't know how to read the documentation, please help.

### ISSUE THREAD

**Issue by vdanyliu** [cvat-ai/cvat#7526]
### Actions before raising this issue

- [X] I searched the existing issues and did not find anything similar.
- [X] I read/searched [the docs](https://github.com/cvat-ai/cvat/tree/master#documentation)

### Is your feature request related to a problem? Please describe.

I don't understand how to use the cvat_sdk for python to create a job or a task... with a hundred images, each of which should have a preliminary annotation (for example, a rectangle with relative coordinates 0.0 0.0 0.5 0.5)

### Describe the solution you'd like

Add more example to doc pls...

### Additional context

![image](https://i.pinimg.com/236x/0f/c7/a0/0fc7a06b9cef1d346838688755df7635.jpg)

**Comment by Oshangsurtani** [comment]
use intellij idea for using cvat_sdk with python

**Comment by vdanyliu** [comment]
> use intellij idea for using cvat_sdk with python
Thank you for the advice, but I'm already doing that and it's not helping much. I've been digging into the low-level API for 2 days and haven't found anything that could help me with this issue.

**Comment by OmarHassan2334** [comment]
I think this code example should be able to help you, from cvat import Dataset, Task

# Assuming you have a list of image paths
image_paths = ['image1.jpg', 'image2.jpg', 'image3.jpg', ...]

# Initialize a dataset
dataset = Dataset()

# Add images to the dataset
for path in image_paths:
    dataset.add_data_entry(path)

# Create a task with the dataset
task = Task(name='My Task', data=dataset)

# Add annotations to each image (e.g., rectangles with relative coordinates)
for data_entry in task.data:
    data_entry.add_annotation("rectangle", points=[0.0, 0.0, 0.5, 0.5])

# Save the task
task.save('my_task.json')


Let me if this works for you and tell me if you need anything else. (I'm commenting this here for easier refence on your behalf I could upload to docs but i need your feedback first.)

**Comment by vdanyliu** [comment]
Alright, thank you.
And what if each image has unique rectangle coordinates?

**Comment by vdanyliu** [comment]
Well... from which library specifically is import Dataset, Task, etc. made? And if it's not cvat_sdk, could you provide a link to the library?

**Comment by zhiltsov-max** [comment]
Hi,

Please start from the [high-level SDK documentation](https://opencv.github.io/cvat/docs/api_sdk/sdk/highlevel-api/) - it begins with a relevant example and provides information on how to perform specific actions. If you want to use lower-level APIs, you should start with [this documentation page](https://opencv.github.io/cvat/docs/api_sdk/sdk/lowlevel-api/), which contains an equivalent relevant example. For annotation uploading, please check [this structure](https://opencv.github.io/cvat/docs/api_sdk/sdk/reference/models/labeled-data-request/) (for the data structure required) - it's used both in high- and low-level APIs.

Please also check SDK tests - there you can find examples for (almost) all high-level APIs provided. Specifically, for the problem you are asking about, these tests should be helpful: [task creation](https://github.com/opencv/cvat/blob/a9333dbe70091ccf65b0fc5a2292a8a939f4bfb3/tests/python/sdk/test_tasks.py#L96), [custom annotation uploading](https://github.com/opencv/cvat/blob/a9333dbe70091ccf65b0fc5a2292a8a939f4bfb3/tests/python/sdk/test_tasks.py#L445) and [annotation patching](https://github.com/opencv/cvat/blob/a9333dbe70091ccf65b0fc5a2292a8a939f4bfb3/tests/python/sdk/test_tasks.py#L497).

Basically, you're looking for a code like this:
```python
from cvat_sdk import make_client

with make_client(...) as client:
    task = client.tasks.create_from_data(...)
    task.update_annotations({"shapes": [{"type": "rectangle", ...}]})
```
